lumen/ai/ui.py

```diff
@@ -347,6 +347,9 @@ class ExplorerUI(UI):
 
     title = param.String(default='Lumen Explorer', doc="Title of the app.")
 
+    chat_loc = param.Selector(default='left', objects=['left', 'right'], doc="""
```

Comment:
Would use a more descriptive name here. Also PLEASE keep parameter definitions sorted unless there's a compelling reason not to.